dandi/cli/tests/test_download.py

```diff
@@ -105,6 +106,10 @@ def test_download_gui_instance_in_dandiset(mocker):
     )
 
 
+@pytest.mark.skipif(
+    bool(known_instances["dandi-api-local-docker-tests"].gui),
+    reason="this instance now has GUI URL",
+)
```

Comment:
If you change the URL in the test to `http://localhost:8085/#/dandiset/123456/draft`, the test will pass and there won't be a need for the `skipif`.

Reply:
;-) that is what I pretty much did first but then looked at the name of the test : `test_download_api_instance_in_dandiset`. And then (copy pasted?) test above differing only in name `test_download_gui_instance_in_dandiset` and url to point to GUI.  So, it is that test above which already checks for that code path and this one was intended to check that we would resolve to api instance if there is no GUI url.  As such, there is no need for two identical tests, and I decided to skip it while keeping for future happen we kill gui there again or smth like that